src/app/archive/archive_lib/processor.ml

```diff
@@ -1071,6 +1068,7 @@ module Block = struct
                 let fee_transfers =
                   Mina_base.Fee_transfer.to_numbered_list fee_transfer_bundled
                 in
+                (* TODO: Need a better name for fee_transfer_ids now that receiver_pk was added to it? *)
```

Comment:
Need to pick something here, maybe just `fee_transfer_infos` or `fee_transfer_tuples`?

Reply:
`fee_transfer_infos` sounds good to me